### PR TITLE
Button

### PR DESCRIPTION
**Pull request by AswathyVNair**

### CODE REVIEW

**Review by sanjitsarkar — APPROVED** [review]
Looks good

File: button/button.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut at orci at
+            justo imperdiet mollis nec vitae erat. Curabitur commodo sapien non
+            diam pharetra vehicula. Praesent efficitur neque nec eros volutpat
+            facilisis.
+          </p>
+
+        <div class="btn-container">
+            
+          <button class="btn-item btn-primary btn-flex btn-primary-1">Primary</button>
```
Comment (humzahasan) [inline button/button.html]: Try writing more descriptive class name. 

primary-1,primary-2 is little difficult to comprehend, as a reviewer.

**Review by humzahasan — APPROVED** [review]
Good to merge, consider the review comments.

**Review by bossmonkey98 — APPROVED** [review]

File: button/button.css [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+.btn-home-icon:hover,
+.btn-bag-icon:hover{
+    color: var(--dark-green);
+}
+
+.btn-heart-icon,
+.btn-search-icon:hover,
+.btn-primary-3{
+    border-radius: 0.5rem;
+}
```
Comment (bhushannaikaroskar) [inline button/button.css]: I guess here you should use values in px as it is less than 10px

File: button/button.html (hunk 1 above)
Comment (bhushannaikaroskar) [reply button/button.html]: Agreed to humza also unable to comprehend where its just color change or something different.

**Review by bhushannaikaroskar — APPROVED** [review]
Some changes suggested. I would suggest to make changes for classes to make it more clear of what they do before merging.

File: button/button.html (hunk 1 above)
Comment (AswathyVNair) [reply button/button.html]: I have made the changes.Thanks for review

File: button/button.css (hunk 2 above)
Comment (AswathyVNair) [reply button/button.css]: Ok, I have changed that to px

**Pull request merged**